### DISCUSSION TITLE
posts/2022-07-01-v010-of-dbt-uaparser-released/

### DISCUSSION THREAD

**Discussion by giscus** [andrewcstewart/andrewcstewart.github.io#9] [Comments]
# posts/2022-07-01-v010-of-dbt-uaparser-released/

User-Agent strings encode information about users that visit sites.  This `dbt` package implements a User Defined Function that leverages UA parsing code that can be used within database queries.

https://www.andrewcstewart.com/posts/2022-07-01-v010-of-dbt-uaparser-released/

**Comment by andrewcstewart** [comment]
Just testing out the giscus comment system.

**Comment by metatheory-org** [comment]
Can you provide an example of how to use this in a dbt python model?
